src/cls/IPM/Lifecycle/Base.cls

```diff
@@ -741,7 +741,16 @@ Method InstallPythonRequirements(
         }
         set tSC = ##class(%IPM.Utils.Module).RunCommand(pRoot, command,.stdout)
         $$$ThrowOnError(tSC)
-
+        set wheelsPath = ##class(%File).NormalizeFilename("wheels", pRoot)
```

Comment:
Should this be moved to be part of HandlePythonDependencies instead? Right now, all wheels would always be installed but we only need them during packaging